marc.go

```diff
@@ -33,8 +34,9 @@ func RetrieveRules(rulefile string) ([]*Rule, error) {
 }
 
 type marcparser struct {
-	file  io.Reader
-	rules []*Rule
+	file          io.Reader
+	rules         []*Rule
+	languageCodes map[string]string
```

Comment:
Since the language codes are only relevant for one piece of information, I wonder if we should just load them where we need them rather than passing them around at the parser level.